Added Doom Emacs

niyumard opened the pull request.

niyumard committed:
Added Doom Emacs

picandocodigo commented:
Hi @niyumard, thank you for this Pull Request!

I think adding Doom Emacs is a great idea. However, can you change the description so that it describes Doom Emacs on its own and not compared to Spacemacs? It can say something like "distribution that aims to be fast and lightweight", or something similar.

Thank you!

niyumard committed:
Change doom emacs description

niyumard committed:
Changing emacs movies with Emacs Series

niyumard commented:
> Hi @niyumard, thank you for this Pull Request!
> 
> I think adding Doom Emacs is a great idea. However, can you change the description so that it describes Doom Emacs on its own and not compared to Spacemacs? It can say something like "distribution that aims to be fast and lightweight", or something similar.
> 
> Thank you!

I also removed the outdated Emacs Movies and replaced it.
Thanks

picandocodigo commented:
Thank you @niyumard!
